include/baresip.h

```diff
@@ -464,6 +471,14 @@ typedef int  (auplay_alloc_h)(struct auplay_st **stp, const struct auplay *ap,
 			      struct auplay_prm *prm, const char *device,
 			      auplay_write_h *wh, void *arg);
 
+/** Defines an Audio Player */
+struct auplay {
+	struct le        le;
+	const char      *name;
+	struct list      dev_list;
```

Comment:
please remove `dev_list` from this patch, so that this patch has no functional
changes; it just moves the struct.